### ISSUE TITLE
Add Sector and Industry to get_stock_quote_type_data?

### ISSUE THREAD

**Issue by austin-stocks** [JECSand/yahoofinancials#32]
I am really enjoying the package and it works as designed. 
This is a request for a feature - I would like to suggest that having a sector and industry for a stock would be a great addition to the method - say get_stock_quote_type_data?

**Comment by JECSand** [comment]
@austin-stocks 

Thanks! I will be happy to look into adding this feature. My goal is to have a new version addressing the current issues out by the middle of September at the latest.

**Comment by jidicula** [comment]
Hi @JECSand, would it be alright if I submitted a PR for this issue? I was trying to build something similar and implemented this feature for my own use before finding this package. It should be simple for me to copy over the functionality.

**Comment by rsandx** [comment]
I second that idea. This is really a great package, but the Profile and Holders pages contain important information that I can't find in existing methods, it'd be great if the data is added in existing or new methods. Keep up the good work and I'm looking forward to the next release!

**Comment by austin-stocks** [comment]
Yes - I agree with rsandx idea of having some way of getting the Profile, number of employess, Key Executives (the whole table). That would be super - as I look at the salary of key executive and how many key executives are leaving the company etc, is the #of employee increasing or decreasing. Thanks for all the work that you do.

**Comment by ivanKrasovec** [comment]
I just wanted to check up on this. I have been using YF and it's been functioning well. I have recently tried implementing get_profile_summary(), which returned `AttributeError: 'YahooFinancials' object has no attribute 'get_profile_data'`
Directly implementing get_stock_tech_data('summaryProfile') returns {'ticker' : None}. I would like to automatically pull which sector each stock belongs to. Any other tips for this issue?
Thank you!
